css/src/components/media.scss

```diff
@@ -0,0 +1,30 @@
+.media {
+	display: flex;
+	align-items: flex-start;
+
+	+ .media,
+	.media {
+		border-top: 1px solid $border;
+		margin-block-start: 1rem;
+		padding-block-start: 1rem;
+	}
+}
+
+.media-left,
+.media-right {
+	flex: 0 0 auto;
+}
+
+.media-left {
+	margin-inline-end: 1rem;
+}
+
+.media-right {
```

Comment:
Should we have a demo example with media on the right side since the class exists?

Reply:
You know, I think we should just remove it